src/Catel.Fody/ModuleWeaver.cs

```diff
@@ -205,10 +205,7 @@ private static Action<string> CreateLoggingCallback(Action<string> callback)
             {
                 Trace.WriteLine(s);
 
-                if (callback != null)
-                {
-                    callback(s);
-                }
+                callback?.Invoke(s);
```

Comment:
Always wondered about this. Isn't this making things slower since it's actually calling .Invoke instead of actually invoking via `callback(s)` or does that result in the same generated (IL) code?